desc/grid.py

```diff
@@ -203,7 +203,10 @@ def _scale_weights(self):
         temp_spacing = self.spacing.copy()
         temp_spacing = (temp_spacing.T / duplicates ** (1 / 3)).T
         # scale weights sum to full volume
-        temp_spacing *= (4 * np.pi**2 / temp_spacing.prod(axis=1).sum()) ** (1 / 3)
+        if temp_spacing.prod(axis=1).sum():
```

Comment:
When would this ever be zero?

Reply:
It happens if the grid is empty for some reason, which occurs in some of our tests:
https://github.com/PlasmaControl/DESC/actions/runs/5583151692/jobs/10203162768#step:6:191